python/src/aac/plugins/gen_plant_uml/gen_plant_uml_impl.py

```diff
@@ -54,7 +62,7 @@ def generate_component_diagram(models: dict):
         return result
 
 
-def puml_sequence(architecture_file: str, output_directory: str = None) -> PluginExecutionResult:
+def puml_sequence(architecture_file: str, output_directory: Union[str, None] = None) -> PluginExecutionResult:
```

Comment:
You can use Optional here instead.
```suggestion
def puml_sequence(architecture_file: str, output_directory: Optional[str] = None) -> PluginExecutionResult:
```